google/cloud/spanner/benchmarks/multiple_rows_cpu_benchmark.cc

```diff
@@ -232,6 +242,86 @@ class SimpleTimer {
 #endif  // GOOGLE_CLOUD_CPP_HAVE_GETRUSAGE
 };
 
+struct BoolTraits {
+  using native_type = bool;
+  static std::string SpannerDataType() { return "BOOL"; }
+  static std::string TableSuffix() { return "bool"; }
+  static native_type MakeRandomValue(
+      google::cloud::internal::DefaultPRNG& generator) {
+    return std::uniform_int_distribution<int>(0, 1)(generator) == 1;
+  }
+};
+
+struct BytesTraits {
+  using native_type = cs::Bytes;
+  static std::string SpannerDataType() { return "BYTES(1024)"; }
+  static std::string TableSuffix() { return "bytes"; }
+  static native_type MakeRandomValue(
+      google::cloud::internal::DefaultPRNG& generator) {
+    std::string tmp = google::cloud::internal::Sample(
+        generator, 1024, "#@$%^&*()-=+_0123456789[]{}|;:,./<>?");
```

Comment:
Opt: Perhaps the `population` here should include all `char`s, to emphasize the nature of `Bytes`?

Reply:
Done.